pcapgo/read.go

```diff
@@ -101,6 +101,9 @@ func (r *Reader) ReadPacketData() (data []byte, ci gopacket.CaptureInfo, err err
 	}
 
 	var n int
+	if 16+ci.CaptureLength > len(r.buf) {
+		return
```

Comment:
+1 to providing that information in the error.

There are times that gopacket telling me that a packet is invalid is perfectly normal.